src/Microsoft.Health.Fhir.Core/Features/Conformance/CapabilityStatementBuilder.cs

```diff
@@ -340,10 +330,8 @@ public ICapabilityStatementBuilder PopulateDefaultResourceInteractions()
             return this;
         }
 
-        public async Task<ICapabilityStatementBuilder> SyncSearchParametersAsync(CancellationToken cancellationToken)
+        public ICapabilityStatementBuilder SyncSearchParametersAsync()
```

Comment:
Low priority: This one should be renamed as well. The async suffix causes doubts.